qemu/tests/cfg/numa_opts.cfg

```diff
@@ -18,84 +20,47 @@
           numa_nodes = 1
     - nodes.1:
       numa_nodes = 1
-      extra_params += " -numa node"
-      numa_node0_cpus = "0 1 2 3 4 5"
+      mem_devs = "mem0"
```

Comment:
How about using the same name format `memN0`?

Reply:
It's just an ID. I used memN0 in hotplug_mem.cfg to be distinguished from the hotplugged ones. In this case, there is no such problem, so I'd like to keep it simple. Thanks.

Reply:
Yes, it is just ID.  I just though with a `N` can mean it is a mem dev for guest_numa_node,  Just personal opioion, it is up to you ~